tests/test_exceptions.py

```diff
@@ -17,11 +17,23 @@
 from conda.base.context import context, conda_tests_ctxt_mgmt_def_pol
 from conda.common.compat import on_win
 from conda.common.io import captured, env_var
-from conda.exceptions import BasicClobberError, BinaryPrefixReplacementError, CommandNotFoundError, \
-    CondaHTTPError, CondaKeyError, CondaRevisionError, DirectoryNotFoundError, \
-    KnownPackageClobberError, ChecksumMismatchError, PackagesNotFoundError, PathNotFoundError, \
-    SharedLinkPathClobberError, TooFewArgumentsError, TooManyArgumentsError, \
-    UnknownPackageClobberError, conda_exception_handler, ExceptionHandler
+from conda.exceptions import (
+    BasicClobberError,
+    BinaryPrefixReplacementError,
+    CommandNotFoundError,
+    CondaHTTPError,
+    CondaKeyError,
+    DirectoryNotFoundError,
+    KnownPackageClobberError,
+    ChecksumMismatchError,
+    PackagesNotFoundError,
+    PathNotFoundError,
+    SharedLinkPathClobberError,
+    TooManyArgumentsError,
+    UnknownPackageClobberError,
+    conda_exception_handler,
+    ExceptionHandler,
+)
```

Comment:
Thank you for consistently improving the readability of the codebase via your PRs!!